test/externalTests/solc-js/DAO/Token.sol

```diff
@@ -23,7 +23,7 @@ corresponding approval process. Tokens need to be created by a derived
 contract (e.g. TokenCreation.sol).
 
 Thank you ConsenSys, this contract originated from:
-https://github.com/ConsenSys/Tokens/blob/master/Token_Contracts/contracts/Standard_Token.sol
+https://github.com/ConsenSys/Token-Factory/blob/master/contracts/StandardToken.sol
```

Comment:
Are you sure this is the right link or are you assuming just based on the names?

Comparing the earliest versions I can find in the repos, they're quite different in terms of implementation - different modifiers, events, inheritance structure:
- https://github.com/ethereum/solidity/blob/8adde5abbe13b015b625862e3657d109ae6a48b5/test/externalTests/solc-js/DAO/Token.sol
- https://github.com/Consensys/Token-Factory/blob/a23e2d9b146a95915dea2b04b087efc1f50896b8/contracts/Standard_Token.sol

If it's not certain, I'd rather leave the link as is, even if it's broken. It's more accurate since that's where the code came from and will be more helpful than a wrong link if anyone ever needs to track down the actual source.